### ISSUE TITLE
hfst-calculate prefixes forward slash to included lexicon file in a SFST

### ISSUE THREAD

**Issue by santhoshtr** [hfst/hfst#370]
My pronouns.fst is:
```
$NSTEM$ = "pronouns.lex" <prn><RB>

$NSTEM$
```

pronouns.lex is a simple text file with a couple of words per each line.

```
$ hfst-calculate -v -i pronouns.fst -o pronouns.a
Output format not specified, defaulting to openfst tropical

reading words from /pronouns.lex...
Error: Cannot open file "/pronouns.lex"!
```

OS is Ubuntu 17.04. Note the compiler prefixes / with file name. This prefixing is coming from https://github.com/hfst/hfst/blob/master/libhfst/src/parsers/SfstCompiler.cc#L638 which says it is for Windows. But surely it get executed in Linux too.

I found one more issue: If I edit the pronouns.fst to have the lex file something else, for example:

```
$NSTEM$ = "somethingelse.lex" <prn><RB>

$NSTEM$
```

I get this error:

```
$ hfst-calculate -v -i pronouns.fst -o pronouns.a
Output format not specified, defaulting to openfst tropical

reading words from /pronouns.lex...
Error: Cannot open file "/pronouns.lex"!
```

It is using the lex file name from the filename of fst.

**Comment by santhoshtr** [comment]
This issue happens only when I compile and use hfst from the released tags. WIth hfst in ubuntu 17.04, all works fine with no issues. That is hfst 3.9.0 though.

**Comment by santhoshtr** [comment]
I bisected till 5bcb93dbe33674c0cdcce225a04d4d8b4acef694
All works till there, may be a few more commits. 
+cc @eaxelson 
I think this is a serious regression for SFST support.

**Comment by eaxelson** [comment]
Fixed the extra '/' issue in commit c14bf89. Does it work now? I cannot replicate the other bug with wrong filename.
